WordPress/Classes/Categories/UIImageView+Gravatar.m

```diff
@@ -81,6 +83,13 @@ - (NSURL *)gravatarURLForEmail:(NSString *)email withSize:(NSInteger)size gravat
     return [NSURL URLWithString:gravatarUrl];
 }
 
+- (NSURL *)siteIconURLForSiteIconUrl:(NSString *)path
+{
+    NSInteger size = [self sizeForBlavatarDownload];
+    NSString *siteIconUrl = [NSString stringWithFormat:@"%@?w=%d&h=%d", path, size, size];
```

Comment:
Hmm... Is that even a possibility, assuming there is no developer errors?

I guess we can add some kind of string check, but I am not sure if that'd be a healthy approach. What do you think?